packages/react/README.md

```diff
@@ -0,0 +1,27 @@
+# @douro-ui/react
+
+Douro UI is a Metyis Design System with React components to be used in projects.
```

Comment:
Provide a description more focused on this package itself, that returns hooks, providers, HOC's, etc...